### ISSUE TITLE
Bug dark mode with timestamps 

### ISSUE THREAD

**Issue by KaaaooS** [richardhbtz/soundcloud-rpc#17]
Timestamps background is in white color and report button too

**Comment by KaaaooS** [comment]
You can update the dark mode for rhe comments and another things buged??

**Comment by billbeans** [comment]
Dark mode doesn't work at all for me

**Comment by KaaaooS** [comment]
at least 1.0.8 still the issue with the dark mode,still not works fine

**Comment by richardhbtz** [comment]
send a screenshot couldnt find those

**Comment by KaaaooS** [comment]
see the holes in white and that windows in white too,im on the 0.0.9 version @richardhbtz 

![Image](https://github.com/user-attachments/assets/03951d78-93b4-48c0-adbc-1fee942b0e2f)

**Comment by richardhbtz** [comment]
guess that only happens when you use other languages, ill have a look into that

<img width="196" alt="Image" src="https://github.com/user-attachments/assets/84f40822-03d8-4db2-8ffc-d4b68b0fe090" />

**Comment by KaaaooS** [comment]
yeah,for i see thats the problem,with the timestamps happens too i dont wknow ifs that is a general bug or not,and that windows with "fans" (thats new lol)
